cli.js

```diff
@@ -34,6 +36,15 @@ files.forEach((file) => {
     } else {
       fs.writeFileSync(file, sorted, 'utf8')
       console.log(`${file} is sorted!`)
+      const diff = Diff.diffLines(packageJson, sorted)
+      diff.forEach((part) => {
+        const colorValue = part.added
+          ? pc.green(part.value)
+          : part.removed
+          ? pc.red(part.value)
+          : pc.gray(part.value)
```

Comment:
Put `+`/`-` before the line? For environment doesn't support color.

Reply:
@fisker 

> Put `+/-` before the line? 

I agree. +/- is now displayed before the line.

## Result

Sorting items

<img width="662" alt="Sorting items" src="https://user-images.githubusercontent.com/26462659/192879246-1d75d8f3-4e87-4fdd-9dcc-89604a4cc4ba.png">

If there are unnecessary row

<img width="651" alt="If there are unnecessary row" src="https://user-images.githubusercontent.com/26462659/192878785-1f9241df-55a4-4892-9e93-264a61a32e74.png">

Reply:
Forgot to commit?

Reply:
Sorry...
I will contact you again when I push commit.

Reply:
Please also add a ` ` for lines without a `-` or `+` so that the file is all inline.

Reply:
Should we add space after `+`?

Reply:
package.json are special normally they are already indented.

Reply:
> Forgot to commit?

I've pushed the current one.
I will check again on the comments I received.